### PR TITLE
fixed barbershop review

### PR DESCRIPTION
**Pull request by taryllngam**
Modifications made on the barbershop website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -175,8 +175,8 @@ <h1>Visit Shop Today</h1>
     <div class="footer-row">
         <div class="footer-left">
             <h1>Opening Hours</h1>
-            <p>Monday To Friday-  9am to 9pm</p>
-            <p>Saturday and Sunday - 8am to 11am</p>
+            <p>Monday To Friday - 7am to 9pm</p>
+            <p>Saturday and Sunday - 9am to 12pm</p>
```
Comment (frckbrice) [inline index.html:179]: why the change?

**Review by frckbrice — CHANGES_REQUESTED** [review]

File: index.html (hunk 1 above)
Comment (taryllngam) [reply index.html:179]: making sure the time is at clients benefits

**Review by taryllngam — COMMENTED** [review]
so i believe the changes will be much more attainable

File: index.html (hunk 1 above)
Comment (frckbrice) [reply index.html:179]: got it
